### PR TITLE
Update neon to 0.9

### PR DESCRIPTION
**Pull request by Theodus**

### CODE REVIEW

**Review by fordN — APPROVED** [review]
Looks good, hope this helps clean up some of the recurring neon build issues. 👍 

I did a quick search on npm for neon and had a few questions around the packages I found called `neon`.

File: Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ keywords = ["neon", "node"]
 
 
 [dependencies]
-neon = "0.8"
+neon = "0.9"
```
Comment (fordN) [inline Cargo.toml:13]: The [neon library at www.npmjs.com](https://www.npmjs.com/package/neon) doesn't appear related to this use case. Is there some naming collisions? 
I would have expected the dep to be https://www.npmjs.com/package/neon-cli.

Comment (Theodus) [reply Cargo.toml:13]: This dependency declaration is for https://crates.io/crates/neon.

**Comment by Theodus** [comment]
I don't have permissions to merge on this repo, so feel free to do so.

File: Cargo.toml (hunk 1 above)
Comment (fordN) [reply Cargo.toml:13]: Ah of course, thanks. 
I have node dependencies on the brain as I've been upgrading all the dependencies in the indexer repo the past few days. 😮‍💨

**Comment by fordN** [comment]
@Theodus I'm in the same boat. 🤔 
@That3Percent ?

**Comment by That3Percent** [comment]
@Theodus @fordN I've just added Edge & Node as a maintainer.
